content/docs/testing-farm.md

```diff
@@ -18,20 +18,33 @@ according to the [Metadata Specification][spec].
 In order to enable test execution simply include `tests` jobs in the `.packit.yaml` configuration:
 
 ```yaml
-    jobs:
-    - job: tests
-      trigger: pull_request
-      metadata:
-        targets:
-        - fedora-all
+jobs:
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
 ```
 
 Required metadata:
-* **targets** - You can use `fedora-development`, `fedora-stable`, `fedora-all`
-  or release specific targets such as `fedora-34-x86_64`.
-  Also `centos-7`, `centos-8` and `centos-stream-8`. The targets are mapped to
-  Testing Farm's [composes](https://api.dev.testing-farm.io/v0.1/composes)
-  when submitting the tests.
+* **targets** - Specify which "builds" you want to test.
+[As with copr_build job](/docs/configuration#available-copr-build-targets) you can use
+specific targets such as `fedora-34-x86_64`. Or just the distro part,
+like `centos-stream-8`, in which case the architecture is `x86_64`. Or an alias,
+like `fedora-development`, `fedora-stable`, `fedora-all` or `epel-all`.
+Each target is then mapped to a [(tmt) distro](https://tmt.readthedocs.io/en/latest/spec/context.html#dimension)
+and to a [Testing farm's compose](https://api.dev.testing-farm.io/v0.1/composes)
+when submitting a test. You can override the default (target to distro) mapping by
+specifying `targets` as a dictionary instead of as a list. In the following example,
+the `fedora-34-x86_64` build will be tested on `centos-stream-8` distro and for
+`epel-7-x86_64` build the default mapping (to `centos-7` distro) will be used:
+```yaml
+  metadata:
+    targets:
+      fedora-34-x86_64:
```

Comment:
can we please map epel-8 into centos-stream-8 which is a more realistic scenario? :)

Reply:
It's so realistic, that it's the default mapping so it makes no sense to have it in the config.
But I can map epel-8 to centos-8